### ISSUE TITLE
Support for go modules

### ISSUE THREAD

**Issue by tsfcode** [ribice/glice#23]
When running glice in a project using go modules, not in the regular gopath you get the error: 
```
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
main.main()
	/home/tommy/go/src/github.com/ribice/glice/main.go:64 +0x758
```

**Comment by ribice** [comment]
Yup, aware of this issue. #9 is the same.

Will need to rewrite large portions of glice to support Go modules, as currently it's tightly coupled to GOPATH.

**Comment by g4s8** [comment]
I'm getting same issue with go-mod app

**Comment by mmcguigan** [comment]
I have the same issue when using glice in a workflow action and found a simple workaround to set the basedir to the fullpath and only call strings.split if the fullpath contains "src/", i.e.

basedir := fullPath
if (strings.Contains(fullPath, "src"+fs)) {
    	basedir = strings.Split(fullPath, "src"+fs)[1]
}

This returned all the same dependencies & license info as the original when run under GOPATH. The only issue is that it also returns the project I ran glice against as a dependency on itself.

**Comment by ribice** [comment]
Resolved with 2eb4a07b38e4b9fdbdce9808773b1de79c1f6574.
